### ISSUE TITLE
Use an static embeder for the web ui files.

### ISSUE THREAD

**Issue by mmarquezs** [xteve-project/xTeVe#122]
**Is your feature request related to a problem? Please describe.**
This is a development side suggestion. I saw the way you embed the UI inside the code and unless I misunderstood the way you are doing it I think you should look into using a tool for that. 

**Describe the solution you'd like**
 Not sure which is currently the best but this are some examples, sorted by their stars in github:
1. https://github.com/gobuffalo/packr <- This one supports to use the locals files while developing. 
1. https://github.com/rakyll/statik
1. https://github.com/mjibson/esc

**Describe alternatives you've considered**
You could leave it as it is. Maybe you have an script for this? But to be honest as files increase it becomes a pain.
It will make it simple to split files in smaller without having to deal with the mess of having to update the webUI.go. For example.... the menu_ts.ts file it's something that clearly should be split in smaller files, separating the classes, and so on.

**Comment by xteve-project** [comment]
I came up with the wild hack many years ago, at that time there were no frameworks to get local files into the Go Binary.

Packr has already been recommended to me. It would be a lot of effort to implement this in xTeVe (Maybe if I have a lot of time)

If you change this line, the local files are used:
xteve.go (Line 51) 
```
const Dev = false
```
The file webUI.go is regenerated with every start to have changes in the later go binary.

The JS files are generated from the Type Script files (TS) with the script compileJS.sh.

Yes you are right, menu_ts.ts has grown a bit :)

**Comment by mmarquezs** [comment]
Well it's nicer than I expected, I thought you manually generated the webUI.go file :P altought it sounded weird.  I already saw the compileJS.sh thing for compiling the typescript files but thanks for the info :) 

This came up cause I wanted to do what I initially thought was an small change to the web UI,... but then I saw that it wasn't as simple change as I expected specially the way the web interface it's build.
That extra info will be helpful for that. 

Feel free to close this or leave it open in case at some point you have a lot of time.
